### PR TITLE
CMake: remove unused packaging script

### PR DESCRIPTION
**Pull request by scivision**
FindSC.cmake was never used by the CMake package, it was always superfluous, so remove it.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.14...3.28)
+cmake_minimum_required(VERSION 3.15...3.28)
```
Comment (elykwilliams) [inline CMakeLists.txt:1]: Why does this commit bump the minimum required version?

**Comment by elykwilliams** [comment]
Why bump the cmake min version number? Am I missing the change that requires 3.15?

**Comment by cburstedde** [comment]
Thanks! Especially the cmakedefine for zlib will be a good thing. Would it be possible to make this PR not from the ECLAIRWaves organization, or to rebase it against @cburstedde's github repository? It has a develop branch that does not match the branch here.

**Comment by cburstedde** [comment]
Ok, using this repository now. Thanks!

**Pull request merged**